pkg/importer/dataStream.go

```diff
@@ -905,13 +916,17 @@ func (d *DataStream) copy(dest string) error {
 			return err
 		}
 	} else {
-		if util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
+		//we do not need scratch space for images with raw format.
+		// d.isIsoImage is assigned to true when it's raw format
+		if !d.isIsoImage && util.GetAvailableSpace(d.ScratchDataDir) <= int64(0) {
```

Comment:
In this path we have already decided we need scratch space in ALL cases, so this makes no sense. Again, simply set d.qemu=true in newDataStream and isHttpQCow2 will be true for an iso image and none of this is necessary.

Reply:
you cannot set d.qemu=true as it will run qemu-img convert on the raw file.

Reply:
So? nothing will happen, doing a convert on a raw file to a raw file, will return a raw file.

Reply:
I do not like to convert something we do need to although there is no harm.
I added changed the code to leave it with two blocks :one that requires scratch space and another that does not.